Looks like a network issue:

```
[Step 2/3] 
[15:07:35][Step 2/3] FAILURE: Build failed with an exception.
[15:07:35][Step 2/3] 
[15:07:35][Step 2/3] * What went wrong:
[15:07:35][Step 2/3] Could not resolve all files for configuration ':compileClasspath'.
[15:07:35][Step 2/3] > Could not resolve com.fasterxml.jackson.core:jackson-databind:2.8.5.
[15:07:35][Step 2/3]   Required by:
[15:07:35][Step 2/3]       project :
[15:07:35][Step 2/3]    > Could not resolve com.fasterxml.jackson.core:jackson-databind:2.8.5.
[15:07:35][Step 2/3]       > Could not get resource 'https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-databind/2.8.5/jackson-databind-2.8.5.pom'.
[15:07:35][Step 2/3]          > Could not GET 'https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-databind/2.8.5/jackson-databind-2.8.5.pom'.
[15:07:35][Step 2/3]             > Connect to repo.maven.apache.org:443 [repo.maven.apache.org/199.232.64.215] failed: connect timed out
```

https://teamcity.cockroachdb.com/viewLog.html?tab=buildLog&logTab=tree&filter=debug&expand=all&buildId=2136050&_focus=575

Whats about just to run tests one more time?

--- a/pkg/sql/logictest/testdata/logic_test/builtin_function
+++ b/pkg/sql/logictest/testdata/logic_test/builtin_function
@@ -2780,3 +2780,8 @@ system
 
 query error pq: crdb_internal.pb_to_json\(\): unknown proto message type cockroach.sql.sqlbase.descriptor
 select crdb_internal.pb_to_json('cockroach.sql.sqlbase.descriptor', descriptor)->'database'->>'name' from system.descriptor where id=1
+
+query B
+SELECT (SELECT * FROM [SHOW crdb_version]) = version()
+----
+true

--- a/pkg/sql/vars.go
+++ b/pkg/sql/vars.go
@@ -914,7 +914,9 @@ var varGen = map[string]sessionVar{
 	},
 
 	// CockroachDB extension.
-	`crdb_version`: makeReadOnlyVar(build.GetInfo().Short()),
+	`crdb_version`: makeReadOnlyVarWithFn(func() string {
+		return build.GetInfo().Short()
+	}),
 
 	// CockroachDB extension
 	`session_id`: {
@@ -1177,6 +1179,13 @@ func makeReadOnlyVar(value string) sessionVar {
 	return sessionVar{
 		Get:           func(_ *extendedEvalContext) string { return value },
 		GlobalDefault: func(_ *settings.Values) string { return value },
+	}
+}
+
+func makeReadOnlyVarWithFn(fn func() string) sessionVar {
+	return sessionVar{
+		Get:           func(_ *extendedEvalContext) string { return fn() },
+		GlobalDefault: func(_ *settings.Values) string { return fn() },
 	}
 }
 

